### DISCUSSION TITLE
[0.1300.0.2]  Beacon station object clipping

### DISCUSSION THREAD

**Discussion by CurePindal** [FakeFishGames/Barotrauma#5311] [Unstable feedback]
The outside antenna/radar on top of the beacon station is clipping though the sub. Probably a draw order issue.

![20210325140306_1](https://user-images.githubusercontent.com/3466193/112483614-7ba67d80-8d79-11eb-94d4-2f8df6760b70.jpg)

**Comment by Krzeszny** [comment]
This is more of an issue than feedback.
